tests/integration/learning/test_discovery_phases.py

```diff
@@ -193,3 +202,7 @@ def find_successful_responses(map_publication_responses):
         # But with debuggers and other processes running on laptops, we give a little leeway.
         assert initial_blocking_duration.total_seconds() < 3
         assert complete_distribution_time.total_seconds() < 10
+
+        # Takes about .8 on jMyles' laptop; still the bulk of init time.
+        # Assuming this is 10 percent of the fleet, that's another 8 second savings for first policy enactment.
+        assert maturation_time.total_seconds() < 2
```

Comment:
pre-commit hooks not running?

Reply:
Yeah, It think I missed a memo - what do I do?